apps/bfd-pipeline/bfd-pipeline-idr/mock-idr.sql

```diff
@@ -339,6 +370,34 @@ CREATE TABLE cms_vdm_view_mdcr_prd.v2_mdcr_clm_line_instnl (
     PRIMARY KEY (geo_bene_sk, clm_dt_sgntr_sk, clm_type_cd, clm_num_sk, clm_line_num)
 );
 
+CREATE TABLE cms_vdm_view_mdcr_prd.v2_mdcr_clm_line_prfnl (
+    geo_bene_sk BIGINT,
```

Comment:
We can mark primary key fields as `NOT NULL`

```suggestion
    geo_bene_sk BIGINT NOT NULL.
```